Sources/Commands/init.swift

```diff
@@ -17,6 +17,16 @@ import func Utility.fputs
 import func Utility.makeDirectories
 import struct Utility.Path
 
+private extension FSProxy {
+    /// Write to a file from a stream producer.
+    mutating func writeFileContents(_ path: String, body: @noescape (OutputByteStream) -> ()) throws {
```

Comment:
Yeah I am trying to gradually kill Utility by making the APIs available on more structured types and with review.